sanic/response.py

```diff
@@ -153,13 +153,26 @@ def html(body, status=200, headers=None):
                         content_type="text/html; charset=utf-8")
 
 
-async def file(location, mime_type=None, headers=None):
+async def file(location, mime_type=None, headers=None, force_download=False):
```

Comment:
you can just pass the `'application/octet-stream'` as mime_type and pass the `"Content-Disposition"` headers.